validator/client/validator.go

```diff
@@ -102,7 +102,6 @@ func (v *validator) WaitForChainStart(ctx context.Context) error {
 			return errors.Wrap(err, "could not receive ChainStart from stream")
 		}
 		v.genesisTime = chainStartRes.GenesisTime
-		break
```

Comment:
Not sure why this was here... Was this intentional @rauljordan ?
https://github.com/prysmaticlabs/prysm/commit/ec9240106e7e466f2cf6a1672e668c0bdbd7d3ca

Reply:
I guess it was because the tests are failing without this statement. I will try to refactor this so that there is no loop any more.